src/main/com/microsoft/azure/documentdb/changefeedprocessor/internal/documentleasestore/DocumentServiceLeaseManager.java

```diff
@@ -644,7 +635,7 @@ public PartitionInfo(String databaseName, String collName, String id, String par
     }
 
     @Override
-    public Date lastExcetution() {
+    public Date lastExecution() {
```

Comment:
as we are relying on java8, we probably should use java8 time package. Date is getting deprecated in favour of the new datetime classes in java8. But again probably not in the scope of this pr. just for your note.

Reply:
Not in this scope...